src/analyses/unassumeAnalysis.ml

```diff
@@ -0,0 +1,276 @@
+(** Unassume analysis. *)
```

Comment:
Maybe add a comment that the purpose of this analysis is to emit appropriate unassume events for the analyses to consume?